src/controllers/reviews.controller.js

```diff
@@ -12,3 +15,13 @@ exports.getReviewById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getReviews = (req, res, next) => {
+  fetchReviews()
+    .then((reviews) => {
+      res.status(200).send({ reviews });
+    })
+    .catch((err) => {
```

Comment:
Do we need a catch here? We aren't expecting any input from the user such as a parametric id or input body so there shouldn't be a need for sad path testing on this endpoint.